### PR TITLE
Pass minimumJavaVersion to RunMojo so that hpi:run works again

### PR DESCRIPTION
**Pull request by jglick**
Corrects a regression in #75, noticed in https://github.com/jenkinsci/external-monitor-job-plugin/pull/18.

### CODE REVIEW

File: src/it/parent-3x/invoker.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # install, not verify, because we want to check the artifact as we would be about to deploy it
 # release.skipTests normally set in jenkins-release profile since release:perform would do the tests
-invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install
+invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install hpi:run
```
Comment (oleg-nenashev) [inline src/it/parent-3x/invoker.properties]: I am not sure it is a good idea to invoke hpi:run in the test. What is an expected behavior?

**Comment by jglick** [comment]
> the test automation approach does not look to be very reliable

Well the test fails with the observed error when the fix is commented out, and passes with the fix in place. Do you have a better idea?

**Comment by jglick** [comment]
Hmm

```
org.apache.maven.plugin.MojoExecutionException: Error preparing the manifest: /var/jenkins_home/plugins/parent-3x.hpl (No such file or directory)
    at org.jenkinsci.maven.plugins.hpi.HplMojo.execute (HplMojo.java:108)
    at org.jenkinsci.maven.plugins.hpi.RunMojo.generateHpl (RunMojo.java:480)
    at org.jenkinsci.maven.plugins.hpi.RunMojo.execute (RunMojo.java:329)
```

is something I do not get locally.

**Comment by jglick** [comment]
Ah, the test fails when the `JENKINS_HOME` environment variable is set…

**Pull request merged**